File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 4/ex4.c
//Busca Sequencial
#include <stdio.h>

main()
{
	printf("Existencia de um numero no vetor\n\n");

	//vars
	int v[20], i, x, cont=0;

	//Entrada
	for( i=0; i<20; i++)
	{
		printf("Digite v[%d]: ", i);
		scanf("%d",&v[i]);
	}

	printf("Digite um valor X: ");
	scanf("%d", &x);

	//Saída
	printf("\n\nPosicoes do numero %d no vetor:\n\n ", x);
	for(i=0; i<20; i++)
	{
		if( v[i] == x )
		{
			cont++;
			printf("%d  ", i);
		}
	}

	if( cont == 0)
		printf("Nao ha ocorrencias do numero %d no vetor", x);
}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 4/src/Ex4.java
//Importando Scanner
import java.util.Scanner;

public class Ex4 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("Busca de um valor em um vetor\n\n");
		
		//vars
		int v[] = new int[20];
		int i, x, posicao=-1;
		
		//Entrada
		for( i=0; i<20; i++)
		{
			System.out.print("Digite v["+i+"]: ");
			v[i] = input.nextInt();
		}
		
		System.out.print("Digite um valor X: ");
		x = input.nextInt();
		
		//Processamento
		for( i=0; i<20; i++)
		{
			if( v[i] == x)
			{
				posicao = i;
			}
		}
		
		//Saída
		if( posicao >= 0 )
		{
			System.out.println("\nValor encontrado na posição "+ posicao);
		}
		else
		{
			System.out.println("Nenhum valor encontrado");
		}
		
	}
}
